.gitignore

```diff
@@ -3,3 +3,8 @@ pom.xml.tag
 pom.xml.releaseBackup
 pom.xml.next
 release.properties
+.DS_Store
+nb-configuration.xml
```

Comment:
Our policy is for IDE-specific files to be added to your “global” gitignore, not in the project .gitignore. See https://github.com/eurekaclinical/dev-wiki/wiki/Development-guidelines-and-requirements for details.